core/shared/gc/ems_gc.h

```diff
@@ -1,15 +1,7 @@
 /*
- * Copyright (C) 2019 Intel Corporation.  All rights reserved.
+ * Copyright (C) 2022 Intel Corporation. All rights reserved.
```

Comment:
Don't rename the folder, the folder structure was discussed and defined before, had better not change it as some users may manually write the Makefile (but not use cmake), and it is not convenient for them to update the Makefile.